avoid including arch.mk, which contains hardcoding settings we don't need/want
author: Miguel Dias Costa (National University of Singapore)
--- Common/common-rules.mk.orig	2016-04-20 13:44:00.254262000 +0800
+++ Common/common-rules.mk	2016-04-20 13:44:07.830742486 +0800
@@ -1,5 +1,5 @@
-$(if $(wildcard $(PREFIX)/arch.mk),,$(error Error: Please create arch.mk from config/ directory for machine-dependent configuration))
-include $(PREFIX)/arch.mk
+#$(if $(wildcard $(PREFIX)/arch.mk),,$(error Error: Please create arch.mk from config/ directory for machine-dependent configuration))
+#include $(PREFIX)/arch.mk
 
 ifneq ($(wildcard $(PREFIX)/flavor.mk),)
   include $(PREFIX)/flavor.mk
